vizro-core/src/vizro/models/_components/form/date_picker.py

```diff
@@ -71,33 +72,15 @@ def build(self):
             output=output,
             inputs=inputs,
         )
-        # clientside callback is required as a workaround when the date-picker is overflowing its parent container
-        # if there is not enough space. Caused by another workaround for this issue:
-        # https://github.com/snehilvj/dash-mantine-components/issues/219
-        clientside_callback(
-            ClientsideFunction(namespace="date_picker", function_name="update_date_picker_position"),
-            output=Output(self.id, "dropdownPosition"),
-            inputs=Input(self.id, "n_clicks"),
-        )
-
-        date_picker_class = dmc.DateRangePicker if self.range else dmc.DatePicker
 
-        # dropdownPosition must be set to bottom-start as a workaround for issue:
-        # https://github.com/snehilvj/dash-mantine-components/issues/219
-        # clearable must be set to False as a workaround for issue:
-        # https://github.com/snehilvj/dash-mantine-components/issues/212
-        # maxDate must be increased by one day, and later on disabledDates must be set as maxDate + 1 day
-        # as a workaround for issue: https://github.com/snehilvj/dash-mantine-components/issues/230
-        date_picker = date_picker_class(
```

Comment:
Saw that all of these issues have been fixed in the mantine repository @AnnMarieW - this is amazing! 🥳 

@nadijagraca @petar-qb - could you double-check manually if all of the issues we've faced initially are solved now? I remember for one of the bugs, we had to add the datepicker to the right side and then change screen sizes etc. I don't know if there were other scenarios like that which need manual checking. You will know best 👍

Reply:
After some testing with various configuration inputs, I can say that all our previous workaround solutions are unnecessary as `dmc.DatePickerInput` handles them all internally 🎉 Here's the example of our workaround solutions.
- maxDate=self.max + datetime.timedelta(days=1) if self.max else None,
- dropdownPosition="bottom-start",
- clearable=False,
- disabledDates=self.max + datetime.timedelta(days=1) if self.max else None,